int8 modem_id
---
float64 slant_range
